examples/app-npm-webpack/webpack-config.js

```diff
@@ -1,12 +1,17 @@
+var path = require('path');
 var webpack = require( "webpack" );
 var CommonsChunkPlugin = require( "webpack/lib/optimize/CommonsChunkPlugin" );
 var HtmlWebpackPlugin = require( "html-webpack-plugin" );
 var GlobalizePlugin = require( "globalize-webpack-plugin" );
-var nopt = require( "nopt" );
 
-var options = nopt({
-	production: Boolean
-});
+var options = {
+	production: (process.env.NODE_ENV === "production"),
+	chunks: [
+		"vendor",
+		"globalize-compiled-data-en",
+		"main"
+	]
```

Comment:
Do we need to define these? This should be created automatically by the plugin, right?

Reply:
yes, I think the plugin should take care of including the chunks in the right order - I just don't know how to do it